.github/workflows/java8_integration_tests.yml

```
@@ -59,6 +59,8 @@ jobs:
           ALLUXIO_DOCKER_NO_TTY=true \
           ALLUXIO_DOCKER_GIT_CLEAN=true \
           ALLUXIO_DOCKER_MVN_TESTS=${{ matrix.modules }} \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE=\!assembly/client,\!assembly/server,\!webui \
+          ALLUXIO_DOCKER_MVN_PROJECT_LIST_TEST=dora/tests/integration \
```

```suggestion
          ALLUXIO_DOCKER_MVN_PROJECT_LIST_COMPILE=\!assembly/client,\!assembly/server,\!webui \
          ALLUXIO_DOCKER_MVN_PROJECT_LIST_TEST=dora/tests/integration \
          ALLUXIO_DOCKER_MVN_TESTS=${{ matrix.modules }} \
```
because this is the order they are used. same for other files